test/specs/commands/createTransactionCastVotes.js

```diff
@@ -811,6 +811,32 @@ describe('create transaction cast votes', () => {
 						);
 					},
 				);
+				Given(
+					'a public key that should be voted for "1234567812345678123456781234567812345678123456781234567812345678"',
+					given.aPublicKeyThatShouldBeVotedFor,
+					() => {
+						Given(
```

Comment:
This might make more sense nested alongside the other options object `Given` steps? (Also applies to other test files.)

Reply:
Hm, I wanted it to be outside of the passphrase scope.

Reply:
Or maybe can you elaborate a bit more?

Reply:
Need to discuss this in person, but this needs to be nested under `Given a Lisk object that can create transactions` at minimum.